### PR TITLE
Setup tests and applied minor fixes for the App Engine Websockets sample

### LINKED ISSUE #1825: build error
I downloaded  websocket samples . 
It is easy to understand to deploy in local . I like it 

and I tried to deploy it by reading [this](https://cloud.google.com/appengine/docs/flexible/nodejs/using-websockets-and-session-affinity?hl=ja)

then  I executed below command
```
gcloud app deploy
```
but , something error happend 
out put is 
```
ERROR: (gcloud.app.deploy) Cloud build failed. Check logs at https ...
UNKNOWN: Error Response: [2] Build failed; check build logs for details

```
so i checked the URL, it said
```
Step #1: yarn install v1.17.3
Step #1: [1/5] Validating package.json...
Step #1: [2/5] Resolving packages...
Step #1: [3/5] Fetching packages...
Step #1: info fsevents@2.1.3: The platform "linux" is incompatible with this module.
Step #1: info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
Step #1: error puppeteer@3.1.0: The engine "node" is incompatible with this module. Expected version ">=10.18.1". Got "10.16.1"
Step #1: error Found incompatible module.
Step #1: info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Step #1: error building image: error building stage: waiting for process to exit: exit status 1
Finished Step #1
ERROR
```

so i change  pacage.json 
before
```
  "engines": {
    "node": ">=8.0.0"
  },
  "scripts": {
```

new
```
  "engines": {
    "node": ">=10.18.1"
  },
  "scripts": {
```

It works well

### PR DESCRIPTION
**Pull request by michaelawyu**
See also cl/315382224.

Fixes #1825.

### CODE REVIEW

File: appengine/websockets/test/index.test.js [hunk 1]
```diff
@@ -15,18 +15,24 @@
 'use strict';
 
 const assert = require('assert');
+const path = require('path');
+const app = require(path.join(__dirname, '../', 'app.js'));
```
Comment (ace-n) [inline appengine/websockets/test/index.test.js:19]: Nit: would `path.dirname(__dirname)` be cleaner/more resilient here?

**Comment by fhinkel** [comment]
The websockets tests are failing.

File: appengine/websockets/test/index.test.js (hunk 1 above)
Comment (michaelawyu) [reply appengine/websockets/test/index.test.js]: Done ;)

**Comment by michaelawyu** [comment]
The tests failed as the Docker image Kokoro uses does not have `libX11-xcb` installed, which is required for Puppeteer to run. The Puppeteer test is disabled for now.

**Comment by fhinkel** [comment]
Wouldn't it make more sense to add the package to the docker image?

**Comment by michaelawyu** [comment]
@fhinkel I do agree and I think the same way. However I am not sure what are the right steps (or more precisely whose approvals should I have for doing this) to update the docker image. I do have access to the container registry but I assume simply pushing a new image version might not be a good idea.

**Comment by fhinkel** [comment]
Can you ask in the DPE chat, usually there's no objections to making changes.

**Comment by michaelawyu** [comment]
Note: A new image `node:10-puppeteer` has been added.

**Comment by michaelawyu** [comment]
All related tests have passed.

**Pull request merged**